rhsm-client/src/main/java/org/candlepin/subscriptions/conduit/rhsm/client/StubRhsmApi.java

```diff
@@ -58,6 +58,7 @@ public OrgInventory getConsumersForOrg(String xRhsmApiAccountID, Integer limit,
         consumer1.getFacts().put("uname.machine", "x86_64");
         consumer1.getFacts().put("virt.is_guest", "True");
         consumer1.getFacts().put("ocm.units", "Sockets");
+        consumer1.getFacts().put("billing_model", "standard");
```

Comment:
```suggestion
        consumer1.getFacts().put("ocm.billing_model", "standard");
```

Reply:
I made the recommended changes, thanks for your help with this.  I will update the testing procedure tomorrow after I run it.